src/explore-education-statistics-frontend/src/pages/cookies/details.tsx

```diff
@@ -185,6 +224,19 @@ function CookiesPage() {
               </td>
             </tr>
           </tbody>
+          <tbody className="govuk-table__body">
+            <tr className="govuk-table__row">
+              <td className="govuk-table__cell">
+                {allowedCookies.disableUserTestingBanner.name}
+              </td>
+              <td className="govuk-table__cell">
+                Disables user testing banner invitation
+              </td>
+              <td className="govuk-table__cell">
+                {allowedCookies.disableUserTestingBanner.duration}
+              </td>
+            </tr>
+          </tbody>
```

Comment:
Doesn't seem like we need this.

Side note - whilst multiple `tbody` elements is permissible, this doesn't look like a case where we would use it. We would just want to add this as another row to the existing `tbody` (if we actually needed it).

Reply:
Cool, this is what it looks like now: 
![Screenshot 2023-03-22 at 11 12 48](https://user-images.githubusercontent.com/55030296/226887572-35b39a40-c7c3-4c1d-a76f-d030f8f78f87.png)